lib/apiAiResponseFormatter.js

```diff
@@ -13,24 +13,36 @@ const contextOut = (shownShelterIds) => (
     ] : []
 );
 
-const responseObject = (text, shownShelterIds = []) => (
+const responseObject = (text, shownShelterIds = [], mapsUrl) => (
   {
     speech: text,
     displayText: text,
+    mapsUrl: mapsUrl,
     contextOut: contextOut(shownShelterIds),
     data: {},
     source: 'irma-api',
   }
 );
 
+const googleMapsUrl = (zipLatLon, shelter) => {
+  let mapsUrlOrigin = zipLatLon.join(',');
```

Comment:
Why is the map origin the zip code? Shouldn't it be the lat/lon of the shelter?

Reply:
Wouldn't the origin be the place the person is starting? The destination would be the lat long of the shelter.

Reply:
Oh, I misunderstood the format of the link -- I had thought that the lat/lon and the address both corresponded to the same point, but actually the lat/lon is the start of directions.

Given that, I'd say a better user experience would be for the starting point to be blank and have Google Maps use the user's current location. It seems unlikely to me that the user would care about directions from the centroid of the ZIP code, since they're probably not there 😛 . It makes the code simpler too...

Reply:
sounds good! i should have all of these comments addressed by tonight. thank you!